pkg/cvo/metrics.go

```diff
@@ -269,6 +278,25 @@ func (m *operatorMetrics) Collect(ch chan<- prometheus.Metric) {
 		g.Set(float64(value))
 		ch <- g
 	}
+
+	installer, err := m.optr.cmLister.Get(internal.InstallerConfigMap)
+	if err == nil {
+		version := "<missing>"
+		invoker := "<missing>"
+
+		if v, ok := installer.Data["version"]; ok {
+			version = v
+		}
+		if i, ok := installer.Data["invoker"]; ok {
+			invoker = i
+		}
+
+		ch <- m.clusterInstaller.WithLabelValues("openshift-install", version, invoker)
+	} else if apierrors.IsNotFound(err) {
+		ch <- m.clusterInstaller.WithLabelValues("", "", "")
+	} else {
+		klog.Warningf("Unable to get installer ConfigMap: %v", err)
```

Comment:
Is this too busy of an error?  We get scraped every 15s, so you'd have a ton of logs in this case.